### PR TITLE
[Merged by Bors] - chore: yet more unused variables

### PR DESCRIPTION
**Pull request by adomani**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Sites/Canonical.lean [hunk 1]
```diff
@@ -44,8 +44,8 @@ variable {C : Type u} [Category.{v} C]
 namespace Sheaf
 
 variable {P : Cᵒᵖ ⥤ Type v}
-variable {X Y : C} {S : Sieve X} {R : Presieve X}
-variable (J J₂ : GrothendieckTopology C)
+variable {X : C}
+variable (J : GrothendieckTopology C)
```
Comment (grunweg) [inline Mathlib/CategoryTheory/Sites/Canonical.lean:48]: You could combine thes einto one line, right?

File: Mathlib/Computability/Reduce.lean [hunk 2]
```diff
@@ -268,7 +268,6 @@ theorem disjoin_le {α β γ} [Primcodable α] [Primcodable β] [Primcodable γ]
 
 variable {α : Type u} [Primcodable α] [Inhabited α]
 variable {β : Type v} [Primcodable β] [Inhabited β]
-variable {γ : Type w} [Primcodable γ] [Inhabited γ]
```
Comment (grunweg) [inline Mathlib/Computability/Reduce.lean:271]: Or combine these two lines.

File: Mathlib/ModelTheory/Complexity.lean [hunk 3]
```diff
@@ -34,11 +34,10 @@ namespace FirstOrder
 
 namespace Language
 
-variable {L : Language.{u, v}} {L' : Language}
-variable {M : Type w} {N P : Type*} [L.Structure M] [L.Structure N] [L.Structure P]
-variable {α : Type u'} {β : Type v'} {γ : Type*}
-variable {n l : ℕ} {φ ψ : L.BoundedFormula α l} {θ : L.BoundedFormula α l.succ}
-variable {v : α → M} {xs : Fin l → M}
+variable {L : Language.{u, v}}
```
Comment (grunweg) [inline Mathlib/ModelTheory/Complexity.lean:37]: You could combine these lines (up to you).

File: Mathlib/ModelTheory/Syntax.lean [hunk 4]
```diff
@@ -61,7 +61,7 @@ namespace FirstOrder
 namespace Language
 
 variable (L : Language.{u, v}) {L' : Language}
-variable {M : Type w} {N P : Type*} [L.Structure M] [L.Structure N] [L.Structure P]
+variable {M : Type w}
 variable {α : Type u'} {β : Type v'} {γ : Type*}
```
Comment (grunweg) [inline Mathlib/ModelTheory/Syntax.lean:65]: Combine this line with the previous one?

**Review by grunweg — COMMENTED** [review]
Thanks for continuously pushing on this. I only have style nits, which are optional anyway.
maintainer merge

**Review by jcommelin — COMMENTED** [review]
bors d+

**Comment by adomani** [comment]
Thanks!  I combined the lines and will wait for CI to finish before merging!

**Comment by adomani** [comment]
bors merge

**Pull request closed without merging**